src/max_encoded_len.rs

```diff
@@ -15,6 +15,8 @@
 
 //! `trait MaxEncodedLen` bounds the maximum encoded length of items.
 
+#![cfg(feature = "max-encoded-len")]
```

Comment:
This is redundant with the feature gate on `mod max_encoded_len;` in `lib.rs`; it should be fine to get rid of it.

Reply:
Done in https://github.com/paritytech/parity-scale-codec/pull/282/commits/5c022a4250244afd0294b6963640184884ac3cbd